From c6df67468bc9f9ae6b0e06aedd158f7cc4a5c6e9 Mon Sep 17 00:00:00 2001
From: =?UTF-8?q?Jan-Simon=20M=C3=B6ller?= <dl9pf@gmx.de>
Date: Wed, 14 Nov 2012 10:39:57 +0100
Subject: [PATCH 02/51] Adapt warnings for compilation with clang
MIME-Version: 1.0
Content-Type: text/plain; charset=UTF-8
Content-Transfer-Encoding: 8bit

Author:  PaX Team <pageexec at freemail.hu>
ML-Post: http://lists.cs.uiuc.edu/pipermail/llvm-commits/Week-of-Mon-20120507/142707.html
URL:     http://llvm.linuxfoundation.org

Merge:   Jan-Simon Möller <dl9pf at gmx.de>

Description:
Adapt warnings for compilation with clang
---
 scripts/Makefile.build |    7 +++++--
 1 Datei geändert, 5 Zeilen hinzugefügt(+), 2 Zeilen entfernt(-)

diff --git a/scripts/Makefile.build b/scripts/Makefile.build
index 0e801c3..cf416f2 100644
--- a/scripts/Makefile.build
+++ b/scripts/Makefile.build
@@ -64,12 +64,15 @@ warning-  := $(empty)
 
 warning-1 := -Wextra -Wunused -Wno-unused-parameter
 warning-1 += -Wmissing-declarations
+warning-1 += -Wno-initializer-overrides
 warning-1 += -Wmissing-format-attribute
-warning-1 += -Wmissing-prototypes
+#warning-1 += -Wmissing-prototypes
 warning-1 += -Wold-style-definition
 warning-1 += $(call cc-option, -Wmissing-include-dirs)
-warning-1 += $(call cc-option, -Wunused-but-set-variable)
+#warning-1 += $(call cc-option, -Wunused-but-set-variable)
 warning-1 += $(call cc-disable-warning, missing-field-initializers)
+warning-1 += -Wno-unused-value -Wno-format -Wno-unknown-warning-option -Wno-self-assign
+warning-1 += -fcatch-undefined-behavior -Wno-sign-compare -Wno-format-zero-length -Wno-uninitialized
 
 warning-2 := -Waggregate-return
 warning-2 += -Wcast-align
-- 
1.7.10.4

